lib/ui/views/edit_dish/edit_dish_view.dart

```diff
@@ -0,0 +1,128 @@
+              style: typography?.titleBold16?.copyWith(
+                color: palette?.gray12,
+              ),
+            ),
+          ),
+          body: SingleChildScrollView(
+            child: Padding(
+              padding: EdgeInsets.symmetric(horizontal: 16.0.w),
+              child: Column(
+                children: [
+                  Text(
+                    'Kindly edit the details below to the reflect the right standard dish information',
+                    style: typography?.labelRegular14?.copyWith(
+                      color: palette?.gray8,
+                    ),
+                  ),
+                  verticalSpaceMedium,
+                  TextFormField(
+                    decoration: InputDecoration(
+                      labelText: 'Name of Dish',
+                      hintText: 'Name of Dish',
+                    ),
+                  ),
+                  16.verticalSpace,
+                  TextFormField(
+                    maxLines: null,
+                    decoration: InputDecoration(
+                      labelText: 'Instructions',
+                      hintText: 'Instructions',
+                    ),
+                  ),
+                  16.verticalSpace,
+                  TextFormField(
+                    controller: ingredientsController,
+                    textInputAction: TextInputAction.done,
+                    onFieldSubmitted: (input) =>
+                        viewModel.addTextFieldInputToList(input),
+                    autocorrect: false,
+                    decoration: InputDecoration(
+                      labelText: 'Ingredients',
+                      hintText: 'Ingredients',
+                    ),
+                  ),
+                  verticalSpaceTiny,
+                  AnimatedOpacity(
+                    opacity: viewModel.allIngredients.isEmpty ? 1 : 0,
+                    duration: Duration(milliseconds: 500),
+                    child: Row(
+                      children: const [
+                        Text('Click "Done" after your input.'),
+                      ],
+                    ),
+                  ),
+                  Wrap(
+                    runSpacing: 10,
+                    spacing: 10,
+                    clipBehavior: Clip.antiAlias,
+                    children: [
+                      for (var ingredientIndex = 0;
+                          ingredientIndex < viewModel.allIngredients.length;
+                          ingredientIndex++)
+                        IngredientChip(
+                          content: viewModel.allIngredients[ingredientIndex],
+                          onTap: () => viewModel
+                              .removeIngredientFromList(ingredientIndex),
+                        ),
+                    ],
+                  ),
+                  100.verticalSpace,
+                  PrimaryButton(
+                    buttonText: 'Edit Dish',
+                    onTap: () {},
```

Comment:
create a empty function for this in the view model and fix it in here.

Reply:
okay